win32ss/user/ntuser/window.c

```diff
@@ -1866,6 +1866,10 @@ PWND FASTCALL IntCreateWindow(CREATESTRUCTW* Cs,
    pWnd->ExStyle = Cs->dwExStyle;
    pWnd->cbwndExtra = pWnd->pcls->cbwndExtra;
    pWnd->pActCtx = acbiBuffer;
+
+   if (pti->spDefaultImc && Class->atomClassName != gpsi->atomSysClass[ICLS_BUTTON])
```

Comment:
why button is excluded?

Reply:
`ImmGetContext` returns `NULL` against the buttons.
The button doesn't need the input context.

Reply:
But this is far from being the only control that a priori doesn't need input context...

Reply:
https://jira.reactos.org/secure/attachment/61923/CtrlContext.zip
![image](https://user-images.githubusercontent.com/2107452/163506163-0c666da2-7357-4ffb-9057-7cc547cd7858.png)
The buttton is special.